content/applications/finance/accounting/reporting/declarations/tax_returns.rst

```diff
@@ -59,65 +60,55 @@ grids that are used to record invoices or credit notes.
+   for more information about Fiscal Localization Packages.
 
 Close a tax period
 ==================
 
 Tax Lock Date
 -------------
 
-Any new transaction which accounting date is prior to the **Tax Lock Date** has its tax values moved
-to the next open tax period. This is useful to make sure that no change can be made to a report once
-its period is closed.
+Any new transaction whose accounting date is before to the :guilabel:`Tax Lock Date` has its tax
+values moved to the next open tax period. This is useful to make sure that no change can be made to
+a report once its period is closed.
 
-Therefore, we recommend locking your tax date before working on your *Closing Journal Entry*. This
-way, other users can't modify or add transactions that would have an impact on the Closing Journal
-Entry, which helps you avoid some tax declaration errors.
+Therefore, we recommend locking your tax date before working on your
+:guilabel:`Closing Journal Entry`.
+This way, other users cannot modify or add transactions that would have an impact on the
+:guilabel:`Closing Journal Entry`, which can help you avoid some tax declaration errors.
 
-To check the current **Tax Lock Date**, or to edit it, go to :menuselection:`Accounting -->
-Accounting --> Lock Dates`.
+To check the current :guilabel:`Tax Lock Date`, or to edit it, go to
+:menuselection:`Accounting --> Accounting --> Actions: Lock Dates`.
 
 .. image:: tax_returns/tax_return_lock.png
    :align: center
-   :alt: Lock your tax for a specific period in Odoo Acounting
+   :alt: Lock your tax for a specific period in Odoo Accounting
 
 Tax Report
 ----------
 
 Once all the transactions involving taxes have been posted for the period you want to report, open
-your **Tax Report**.
-
-To do so, go to :menuselection:`Accounting --> Reporting --> Tax Report`. You can also click
-on *TAX Report* from your *Accounting Overview*.
-
+your :guilabel:`Tax Report` by going to :menuselection:`Accounting --> Reporting -->
+Audit Reports: Tax Report`.
 Make sure to select the right period you want to declare by using the date filter. You can see an
-overview of your tax report. Then, click on the button *Closing Journal Entry*.
-
-.. image:: tax_returns/tax_return_closing.png
-   :align: center
-   :alt: Select the period for the tax return and create a closing journal entry in Odoo Accounting
-
-After having reviewed the generated Journal Entry, click on *Post*. In addition to posting the entry,
-Odoo automatically creates a PDF file with the **Tax Report** that you can download from the chatter
-and preview on the right column. It includes all the values to report to the tax authorities, along
-with the amount you have to pay or be refunded.
+overview of your tax report.
+From this view, you can easily access different formats of your tax report, such as `PDF` and
+XLSX.
+These include all the values to report to the tax authorities, along with the amount you have to pay
+or be refunded.
```

Comment:
going to the next line doesn't work in RST (also, it's not something usual in English)
either write a space or make it a new paragraph.